packages/components/src/core/validators/validators.ts

```diff
@@ -1,4 +1,4 @@
-import * as _ from 'lodash';
+const isEmpty = obj => [Object, Array].includes((obj || {}).constructor) && !Object.entries((obj || {})).length;
```

Comment:
Could you please put that in a utility file instead ?

Reply:
Updated!